jest.config.ts

```diff
@@ -0,0 +1,9 @@
+import type {Config} from '@jest/types';
+// Sync object
+const config: Config.InitialOptions = {
+  verbose: true,
+  transform: {
+  '^.+\\.tsx?$': 'ts-jest',
+  },
+};
+export default config;
```

Comment:
```suggestion
import type { Config } from "@jest/types";
// Sync object
const modulesToTransform = [
  "pokedex-promise-v2",
  "p-map",
  "aggregate-error",
  "indent-string",
  "clean-stack",
  "escape-string-regexp",
].join("|");
const config: Config.InitialOptions = {
  verbose: true,
  transform: {
    "^.+\\.(j|t)sx?$": "ts-jest",
  },
  transformIgnorePatterns: [`node_modules/(?!${modulesToTransform}/.*)`],
};
export default config;
```

This configuration let's jest include those packages that apparently were not included properly by default.

In addition you should set as `true` the prop `allowJs` in the `tsconfig.json` file.

Reply:
It worked!! Thank you so much 🙇🏼‍♂️